components/support/nimbus-fml/src/workflows.rs

```diff
@@ -158,6 +160,37 @@ pub(crate) fn fetch_file(files: &LoaderConfig, nm: &str) -> Result<()> {
     Ok(())
 }
 
+pub(crate) fn validate(cmd: &ValidateCmd) -> Result<()> {
+    let files: FileLoader = TryFrom::try_from(&cmd.loader)?;
+
+    let filename = &cmd.manifest;
+    let file_path = files.file_path(filename)?;
+    let parser: Parser = Parser::new(files, file_path.clone())?;
+    let mut loading = HashSet::new();
+    let fe = parser.load_manifest(&file_path, &mut loading)?;
+
+    println!(
+        "Loaded modules: [\n\t{}\n]",
+        loading
+            .iter()
+            .map(|id| id.to_string())
+            .collect::<Vec<String>>()
+            .join(",\n\t")
+    );
+
+    for channel in fe.channels {
+        print!("Validating manifest for channel \"{}\".", &channel);
```

Comment:
You can use a raw string to embed double quotes:
```rust
print!(r#"Validating manifest for channel "{}"."#, &channel);
```

Reply:
Ah cool I need to take some time to learn about those